openhtf/io/output/mfg_inspector.py

```diff
@@ -350,33 +340,20 @@ class OutputToTestRunProto(object):  # pylint: disable=too-few-public-methods
     filename_pattern: A format string specifying the filename to write to,
       will be formatted with the Test Record as a dictionary.  May also be a
       file-like object to write directly to.
-
   Returns:
     filename of local file.
   """
 
   def __init__(self, filename_pattern):
-    self.filename_pattern = filename_pattern
+    super(OutputToTestRunProto, self).__init__(filename_pattern)
 
-  def __call__(self, test_record):  # pylint: disable=invalid-name
-    as_dict = data.ConvertToBaseTypes(test_record)
-    serialized = _TestRunFromTestRecord(test_record).SerializeToString()
-    if isinstance(self.filename_pattern, basestring):
-      if '{' in self.filename_pattern:
-        filename = self.filename_pattern.format(**as_dict)
-      else:
-        filename = self.filename_pattern % as_dict
-      with open(filename, 'w') as outfile:
-        outfile.write(serialized)
-    else:
-      filename = self.filename_pattern
-      self.filename_pattern.write(serialized)
-    return filename
+  @staticmethod
+  def serialize_test_record(test_record):
+    return _TestRunFromTestRecord(test_record).SerializeToString()
 
 
 class UploadToMfgInspector(object):  # pylint: disable=too-few-public-methods
   """Generate a mfg-inspector TestRun proto and upload it.
-
```

Comment:
hm, most of the lines are here intentionally to give visual separation between the short-description and long-description.  can we leave them in?